src/main/java/core/basesyntax/service/CreateReportService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
```

Comment:
This package"core.basesyntax.service" is only for classes with the abbreviation "service"
Rename other classes or move them in appropriate packages.
Also, there should only be a package of impl inside